src/main/java/me/coley/recaf/control/gui/GuiController.java

```diff
@@ -19,7 +22,7 @@
  */
 public class GuiController extends Controller {
 	private final WindowManager windows = new WindowManager(this);
-	private final ConfigManager configs = new ConfigManager();
+	private final ConfigManager configs = new ConfigManager(Recaf.getStorageDirectory().resolve("rc-config"));
```

Comment:
Since this is the only usage of `getStorageDirectory()`, we may as well inline `resolve(rc-config)`. Plus it would make more sense in my opinion that the storage directory has the config folder defined locally.